applications/test_1gb_huge_pages/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "test_1gb_huge_pages"
+version = "0.1.0"
+authors = ["Noah Mogensen <mogensenn@sou.edu>"]
```

Comment:
for this and the other crate, please add a description field too.

Also, can you use a more permanent email address such in case a future contributor needs to get in contact with you?